components/docs.html

```diff
@@ -585,17 +612,80 @@ <h2 class="modal-header">modal Header</h2>
               <i class="fas fa-times"></i>
             </button>
             <div class="modal-body">
-                <p>Lorem ipsum dolor sit amet.</p>
+              <p>Lorem ipsum dolor sit amet.</p>
             </div>
             <div class="modal-footer footer-row">
-                <button class="btn secondary-btn-outlined btn-s modal-btn">Action 1</button><button class="btn secondary-btn-outlined btn-s modal-btn">Action 2</button>
+              <button class="btn secondary-btn-outlined btn-s modal-btn">Action 1</button><button
+                class="btn secondary-btn-outlined btn-s modal-btn">Action 2</button>
             </div>
           </div>
         </div>
       </div>
-      <script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fmohammadkhizerkhan%2Fvite-ui%2Fblob%2Fmodal%2Fcomponents%2Fdocs.html%23L580-594&style=github-gist&showBorder=on&showCopy=on&fetchFromJsDelivr=on"></script>
+      <script
+        src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fmohammadkhizerkhan%2Fvite-ui%2Fblob%2Fmodal%2Fcomponents%2Fdocs.html%23L580-594&style=github-gist&showBorder=on&showCopy=on&fetchFromJsDelivr=on">
+      </script>
       <h3>JS code:</h3>
-      <script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fmohammadkhizerkhan%2Fvite-ui%2Fblob%2Fmodal%2Fjs%2Fmodal.js&style=github-gist&showBorder=on&showCopy=on&fetchFromJsDelivr=on"></script>
+      <script
+        src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fmohammadkhizerkhan%2Fvite-ui%2Fblob%2Fmodal%2Fjs%2Fmodal.js&style=github-gist&showBorder=on&showCopy=on&fetchFromJsDelivr=on">
+      </script>
+    </div>
+    <div id="text-utilities">
+      <h1>Text-Utilities</h1>
+      <h2>To use text align:</h2>
+      <div class="container-column component-container">
+        <h4 class="text-center">Lorem ipsum dolor sit amet </h4>
+        <h4 class="text-left">Lorem ipsum dolor sit amet </h4>
```

Comment:
if possible Try giving different text for different component types

Reply:
sure, thanks